src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35879/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:35879/project/issues?id=sample&branch=feature2&issues=AWDHKqeUbmiBx0h6Bnk_&open=AWDHKqeUbmiBx0h6Bnk_)

[ISSUE_KEY]: AWDHKqeUbmiBx0h6Bnk_